source/includes/fact-valid-namespace-remap.rst

```diff
@@ -0,0 +1,35 @@
+
+The following restrictions apply to namespace remapping:
+
+- Namespace remapping doesn't support changes to collection names.
```

Comment:
Can we remove this bullet? Since we specify changing `database names` in the feature description I don't think we need to list this as a restriction (rather, it's just  not part of the feature at all)

Reply:
Fixed